src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,70 @@
+package duke;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.ArrayList;
+
+/**
+ * Contains methods to write the current tasks state to disk.
+ * 
+ * @author Siau Wee
+ */
+public class Storage {
+
+    private File saveFile;
```

Comment:
```suggestion
    private final File saveFile;
```
Suggestion of a `final` modifier.